CODEOWNERS

```diff
@@ -438,6 +438,7 @@
 /samples/bluetooth/central_nfc_pairing/   @nrfconnect/ncs-si-muffin
 /samples/bluetooth/central_smp_client/    @nrfconnect/ncs-si-muffin
 /samples/bluetooth/central_uart/          @nrfconnect/ncs-si-muffin
+/samples/bluetooth/channel_map_updates/   @nrfconnect/ncs-dragoon
```

Comment:
Please add an entry for the RST file too [here](https://github.com/nrfconnect/sdk-nrf/pull/25235/files/fc8a9bd4cbe170506a4e35ef3ebe6d26c6ba40ca..38d5b8997fd498333b636af8ed5b416d80195b82#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bL574): /samples/bluetooth/channel_map_updates/*.rst   @nrfconnect/ncs-dragoon-doc